spec/license_spec.rb

```diff
@@ -1,35 +1,32 @@
 require 'spec_helper'
 
-# List of licenses that need not be OSI approved
-LICENSE_WHITELIST = %w[
-  unlicense
-  cc0-1.0
-  wtfpl
-  bsd-3-clause-clear
```

Comment:
The whitelist was part of the old test (which only looked at OSI approval), and is being removed by this pull request.